package.json

```diff
@@ -26,6 +26,7 @@
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-eslint": "^3.0.0",
+    "ember-cli-fastboot": "^1.0.2",
```

Comment:
Not an issue but I think you could write the tests in `fastboot-tests`? If it is more against the dummy app, it's fine too.